packages/phoenix-client/src/phoenix/client/resources/datasets/__init__.py

```diff
@@ -1034,6 +1034,103 @@ def _process_dataset_upload_response(
 
         return dataset
 
+    def delete_example_from_dataset(
+        self,
+        *,
+        example_ids: Union[str, List[str]],
+        version_description: Optional[str] = None,
+        version_metadata: Optional[Mapping[str, Any]] = None,
+        return_dataset: bool = False,
+        timeout: Optional[int] = DEFAULT_TIMEOUT_IN_SECONDS,
+    ) -> Optional[Dataset]:
```

Comment:
```suggestion
        timeout: Optional[int] = DEFAULT_TIMEOUT_IN_SECONDS,
    ) -> Dataset:
```

Let's keep the signature and types simpler

Reply:
Probably need to think about the return type if this method is intended to handle multiple deletions at once.

Reply:
It looks like our other `delete` methods have a `None` return type.

Reply:
I see, will do NoneType for now